app_flutter/lib/logic/qualified_task.dart

```diff
@@ -54,23 +56,11 @@ class QualifiedTask {
       case StageName.cirrus:
         return '$_cirrusUrl/master';
       case StageName.luci:
-        return _luciSourceConfigurationUrl;
+        return '$_luciUrl/builders/luci.flutter.prod/$builder';
```

Comment:
This only links to the specific builder page, but I see that task now has additional LUCI information. Is it possible to redirect this to the task specific page now?

Reply:
That has been implemented in https://github.com/flutter/cocoon/pull/820.